common/djangoapps/course_about/tests/test_api.py

```diff
@@ -0,0 +1,4 @@
+"""
+Tests the logical Python API layer of the Course About API.
+
+"""
```

Comment:
Actually, it looks like you've got adequate coverage from the view layer tests.  I'd be okay with deleting this module.